### PR TITLE
Add .travis.yml

### PR DESCRIPTION
**Pull request by NuckChorris**
Hey there, I did the honors of adding a basic `.travis.yml` for this.  All the specs are (predictably) passing as you can see [here](https://travis-ci.org/hummingbird-me/doorkeeper-grants_assertion).

I'd also like to adopt this gem since I'm hoping to use it for our next-generation social login over at @hummingbird-me and it clearly needs a bit more work before it's quite production-ready. #9 especially seems important

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: ruby
+rvm:
+ - 2.0.0
+ - 2.1.7
+ - 2.2.3
```
Comment (tute) [inline .travis.yml]: We don't need to define patch version, let's define 2.0, 2.1 and 2.2. It will build in latest.

**Comment by tute** [comment]
A minor comment, looks great afterwards! Thanks, and happy to add you as contributor so you can work on it.

**Comment by NuckChorris** [comment]
Fixed the version numbers :thumbsup:

**Comment by tute** [comment]
Thank you! :heart:

**Comment by tute** [comment]
You should now have write access to this repository. :clap: :heart: :smiley:

**Pull request merged**